services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_task_utils.py

```diff
@@ -42,12 +41,6 @@ async def apply_observation_cycle(
             can_save=scheduler_data.dynamic_sidecar.were_containers_created,
         )
 
-    # TODO: ANE this can be moved to a handler in the future scheduled
-    # to all the correct cases
-    scheduler_data.dynamic_sidecar.is_available = (
```

Comment:
so if i understand this right,  this is the main change.... the logic is moved as events